src/common/ConfigManager.h

```diff
@@ -25,6 +25,7 @@ extern bool mirroringEnable;
 extern bool parseDebug;
 extern bool speedHack;
 extern bool speedup;
+extern bool gbaLcdFilter;
```

Comment:
Yes, looking after the good old `ConfigManager`, I do that constantly. We need to rip it out and unify the wx config system.

I am **NOT** asking you to do that here. I'm just saying eventually we will need to do this.